### PR TITLE
Customize theme, fix twitter URL, flesh out about page

### PR DESCRIPTION
**Pull request by kburk1997**

### CODE REVIEW

**Comment by aeksco** [comment]
:+1: Nice work!

File: about.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ profile: true
 ---
 
 We are a community of open source developers at Rensselaer.
-This is a static page. It could be an 'about page' if you'd like.
 
-{% include footer.html %}
+Our projects can be found on our open source project managent platform, [Observatory](http://rcos.io).
```
Comment (agundy) [inline about.md]: managent -> management

Comment (agundy) [reply about.md]: and https://rcos.io/ is better than http

File: _includes/footer.html [hunk 2]
```diff
@@ -1,3 +1,4 @@
 <footer id="footer">
-    <p class="small">© Copyright {{ site.time | date: '%Y' }} {{ site.author }}</p>
+    <p class="small">© Copyright {{ site.time | date: '%Y' }} {{ site.author }} | <a href="https://github.com/rcos/rcos.github.io">Github</a></p>
```
Comment (agundy) [inline _includes/footer.html]: might be good to also link to rcos.io since it's the main place to see what rcos is up to

**Review by agundy — CHANGES_REQUESTED** [review]
fix a typo and a better link

File: about.md (hunk 1 above)
Comment (kburk1997) [reply about.md]: Whoops -- didn't catch that lol

**Comment by kburk1997** [comment]
Fixed that typo and that link, works on my local machine

**Review by agundy — APPROVED** [review]

**Pull request merged**